### PR TITLE
wrap in fragment on duplicate

### PR DESCRIPTION
**Pull request by bkrmendy**
## Description
This PR refactors the `duplicate` function so that is uses `insertJSXElementChild` internally, making it compatible with conditional expressions. This PR also adds test coverage for duplicating elements via the duplicate shortcut.

### CODE REVIEW

**Review by ruggi — APPROVED** [review]

File: editor/src/components/canvas/canvas-utils.ts [hunk 1]
```diff
@@ -2816,14 +2818,33 @@ export function duplicate(
...
+            }
+
+            const insertionPath =
+              conditionalCase != null
+                ? conditionalClauseInsertionPath(
+                    EP.parentPath(path),
+                    conditionalCase,
+                    'wrap-with-fragment',
+                  )
+                : childInsertionPath(EP.parentPath(newPath))
```
Comment (ruggi) [inline editor/src/components/canvas/canvas-utils.ts:2838]: I think you could refactor this to benefit from `getInsertionPathWithWrapWithFragmentBehavior`

**Review by Rheeseyb — APPROVED** [review]
